### PR TITLE
Feat: Add missing sidecar and global parameter for deployments & hooks

### PR DESCRIPTION
**Pull request by dromadaire54**
This pull request consists to add global sidecars or volumes to all deployments including hooks. In my case, the same sidecar and volume were added in all deployments and duplicated in the values file. Furthermore, I added the missing sidecar component in the deployments and hooks level. Let me know if you are suggestions about my PR.

### CODE REVIEW

**Comment by dromadaire54** [comment]
@Mokto I don't understand why the chart testing not working. There are errors about clickhouse.

**Comment by Mokto** [comment]
Can you please rebase your branch? A fix has been merged into develop.

**Comment by dromadaire54** [comment]
Sorry @Mokto I rebased my branch but it didn't fix it.

**Comment by Mokto** [comment]
That's fine, it's a different failure that @TartanLeGrand is working on.

**Comment by TartanLeGrand** [comment]
Yes, when have been upgrade correctly the chart to sentry 24.6.0 just rebase your PR 😉

**Comment by dromadaire54** [comment]
Thanks @TartanLeGrand and @Mokto for the answer.

**Comment by Mokto** [comment]
:wave: Hi, @dromadaire54,
I detected conflicts against the base branch :speak_no_evil:
You'll want to sync :arrows_counterclockwise: your branch with upstream!

**Comment by dromadaire54** [comment]
@Mokto I resolved all the conflicts and the test & lint chart working perfectly.

File: sentry/templates/deployment-sentry-ingest-consumer-transactions.yaml
Comment (Mokto) [inline sentry/templates/deployment-sentry-ingest-consumer-transactions.yaml:1]: I'm not sure I understand that change.

Comment (dromadaire54) [reply sentry/templates/deployment-sentry-ingest-consumer-transactions.yaml:1]: ah yes sorry

Comment (dromadaire54) [reply sentry/templates/deployment-sentry-ingest-consumer-transactions.yaml:1]: Done @Mokto

**Pull request merged**